setup.py

```diff
@@ -0,0 +1,33 @@
+from setuptools import setup
+
+
+with open('cherry_picker/readme.rst') as f:
+    long_description = f.read()
+
+
+with open('cherry_picker/requirements.txt') as f:
+    requirements = [l.strip() for l in f.read().split('\n')]
+
+
+setup(
+    name='cherry_picker',
+    description='backport cpython changes from master to maintenance branches',
+    long_description=long_description,
+    license='Apache',
+    url='https://github.com/python/core-workflow',
+    author='Mariatta Wijaya',
+    author_email='mariatta.wijaya@gmail.com',
```

Comment:
Thanks 😸  Well I've prepared a canned response already "Please file an issue at python/core-workflow" 

I'll add the following:
```
"maintainer": "Python Core Developers",
"maintainer_email": "core-workflow@mail.python.org",
```

Question: is `version` metadata not needed? The [doc](https://docs.python.org/3.6/distutils/setupscript.html#meta-data) seems to indicate that it's required.